src/web_app_skeleton/config/cookies.cr.ecr

```diff
@@ -15,6 +15,9 @@ Lucky::CookieJar.configure do |settings|
     # Restrict cookies to a first-party or same-site context
     cookie.samesite(:lax)
 
+    # Set all cookies to the root path by default
+    cookie.path("/")
```

Comment:
Without this, the cookies in the browser had the paths they were set on so if a cookie was added on the `/users` pages, the cookie would be passed along only if you were on that path or further nested (`/users/123/edit`). This was causing multiple session cookies to be in the browser and was not intentional at all.

This only started happening in Crystal 1.0.0 because the default path of `"/"` was removed in https://github.com/crystal-lang/crystal/commit/17f153eae61225957ed67816ab13f6aeac5b7d91

We definitely missed this change but we luckily had testing that caught it 😌